crates/ruff/src/rules/wemake_python_styleguide/helpers/naming.rs

```diff
@@ -0,0 +1,78 @@
+use std::path::Path;
+
+use once_cell::sync::Lazy;
+use regex::Regex;
+
+const ALIAS_NAMES_WHITELIST: [&str; 7] = ["np", "pd", "df", "plt", "sns", "tf", "cv"];
+
+const UNUSED_PLACEHOLDER: char = '_';
+
+static UNUSED_VARIABLE_REGEX: Lazy<Regex> = Lazy::new(|| Regex::new(r"^_+$").unwrap());
+
+/// Checks whether the given ``name`` is unused.
+fn is_unused(name: &str) -> bool {
+    UNUSED_VARIABLE_REGEX.is_match(name)
+}
+
+/// Checks for too short names.
+pub(in crate::rules::wemake_python_styleguide) fn is_too_short_name(
+    name: &str,
+    min_length: usize,
+    trim: bool,
+) -> bool {
+    if ALIAS_NAMES_WHITELIST.contains(&name) {
+        return false;
+    }
+
+    if is_unused(name) {
+        return false;
+    }
+
+    let length = if trim {
+        name.trim_matches(UNUSED_PLACEHOLDER).len()
+    } else {
+        name.len()
+    };
+
+    length < min_length
+}
+
+/// Return `true` if a [`Path`] should use the name of its parent directory as its
+/// module name.
+pub(in crate::rules::wemake_python_styleguide) fn is_module_file(path: &Path) -> bool {
+    path.file_name().map_or(false, |file_name| {
+        file_name == "__init__.py"
+            || file_name == "__init__.pyi"
+            || file_name == "__main__.py"
+            || file_name == "__main__.pyi"
```

Comment:
Nit
```suggestion
        matches!(file_name, "__init__.py" | "__init__.pyi" | "__main__.py" | "__main__.pyi")
```

Reply:
Pattern matching does not seem to work here, because `file_name` is an [OsStr](https://doc.rust-lang.org/std/ffi/struct.OsStr.html). Compiler says:

```
rustc: mismatched types
   expected reference `&std::ffi::OsStr`
      found reference `&'static str` [E0308]
```